aks/extra-variables.tf

```diff
@@ -1,16 +1,8 @@
-variable "storage_account_tier" {
-  description = "Storage account tier used for storing loki logs"
-  default     = "Standard"
-  type        = string
-}
-
-variable "storage_account_replication_type" {
-  description = "Storage account replication type for storing loki logs"
-  default     = "LRS"
-  type        = string
-}
-
-variable "resource_group_name" {
-  description = "The Resource Group where the Managed Kubernetes Cluster should exist."
-  type        = string
+variable "logs_storage" {
```

Comment:
this should be a sensitive var right?

Reply:
We need to deal with this latter, as per our discussion in our channel.